testing/scenario_app/lib/src/platform_view.dart

```diff
@@ -28,6 +28,13 @@ List<int> _to64(num value) {
   return temp;
 }
 
+List<int> _encodeString(String value) {
```

Comment:
This was just a drive-by improvement since I accidentally mismatched the strings in the `.length` and `utf8.encode` lines when I added a new string, so I thought I'd prevent the whole class of mistake once I realized what I'd done wrong.